bindings/nodejs/examples/how_tos/accounts_and_addresses/consolidate-outputs.ts

```diff
@@ -67,9 +67,9 @@ async function run() {
 
         // Consolidate unspent outputs and print the consolidation transaction ID
         // Set `force` to true to force the consolidation even though the `output_consolidation_threshold` isn't reached
```

Comment:
lets add this comment to the `ConsolidationParams.ts` now

Reply:
Done